src/app/components/ConnectorForm/index.tsx

```diff
@@ -42,7 +44,28 @@ function ConnectorForm({
         </div>
         <div className="mt-16 lg:mt-0 lg:w-1/2">
           <div className="lg:flex h-full justify-center items-center">
-            <img src="assets/icons/satsymbol.svg" alt="sat" className="w-64" />
+            {video ? (
+              <div
+                className="flex-1 relative h-0"
+                style={{ paddingBottom: "56.25%" }}
+              >
+                <video
+                  className="absolute t-0 l-0 w-full h-full"
+                  width="1920"
+                  height="1080"
```

Comment:
Is it good to have fixed sizes here? what if the user's screen is different?

Reply:
The width and height are overridden by CSS rules anyway to maintain the aspect ratio, I think we can safely remove them.

Reply:
Regarding different screen sizes: there is a responsive solution in this CSS layout so the video always retains its original aspect ratio. (padding-bottom: 56.25% --> 16:9)